docker/api.env

```diff
@@ -1,3 +1,3 @@
 KAFKA_URL=127.0.0.1:9092
 INFORMIX_HOST=127.0.0.1
-POSTGRES_URL=postgres://coder:mysecretpassword@localhost:5432/projectsdb
\ No newline at end of file
+PROJECTS_API=127.0.0.1:8001/v5
```

Comment:
@maxceem don't we need the other new env variables (AUTH0 related) here?

Reply:
I've removed this file at all, as it is supposed not to be added to repo. We have another sample file https://github.com/topcoder-platform/legacy-project-processor/blob/feature/use-projects-api/docker/sample.api.env#L3

Reply:
@maxceem it seems to fail without ap.env. https://circleci.com/gh/topcoder-platform/legacy-project-processor/21

Reply:
Fixed building process via PR https://github.com/topcoder-platform/legacy-project-processor/pull/5, merged to `dev`. Building and deploying by CI works well now https://circleci.com/gh/topcoder-platform/legacy-project-processor/tree/develop.